src/prefect/server/orchestration/core_policy.py

```diff
@@ -53,13 +55,15 @@
 from prefect.settings import (
     PREFECT_API_TASK_CACHE_KEY_MAX_LENGTH,
     PREFECT_DEPLOYMENT_CONCURRENCY_SLOT_WAIT_SECONDS,
-    PREFECT_TASK_RUN_TAG_CONCURRENCY_SLOT_WAIT_SECONDS,
+    get_current_settings,
```

Comment:
should we switch them all to new settings access?

Reply:
Yeah, I can do that